control_msgs/msg/JointJog.msg

```diff
@@ -0,0 +1,18 @@
+# Used in time-stamping the message.
+Header header
+ 
+# Name list of the joints. You don't need to specify all joints of the
+# robot. Joint names are case-sensitive.
+string[] joint_names
+ 
+# Command to the joints listed in joint_names.
+# The order must be identical.
+# If joint positions, units are rad for revolute joints, meters for linear joints.
+# If joint velocities, units are rad/s or m/s.
+float64[] command
```

Comment:
I don't like to mix displacement (position offset) and target velocites in one command.

Reply:
@ipa-mdl Love it!